docs/Makefile

```diff
@@ -189,4 +189,4 @@ xml:
 pseudoxml:
 	$(SPHINXBUILD) -b pseudoxml $(ALLSPHINXOPTS) $(BUILDDIR)/pseudoxml
 	@echo
-	@echo "Build finished. The pseudo-XML files are in $(BUILDDIR)/pseudoxml."
+	@echo "Build finished. The pseudo-XML files are in $(BUILDDIR)/pseudoxml."
```

Comment:
I was asked to do it by @tsnoam in the [previous pull request](https://github.com/python-telegram-bot/python-telegram-bot/pull/337#issuecomment-231463983).